sbt-scalajs-bundler/src/main/scala/scalajsbundler/sbtplugin/LibraryTasks.scala

```diff
@@ -87,24 +87,21 @@ object LibraryTasks {
         ) { _ =>
           log.info(
             s"Building webpack library bundles for ${entryPointFile.project} in $cacheLocation")
-          Set(
-            Webpack
-              .bundleLibraries(
-                emitSourceMaps,
-                generatedWebpackConfigFile,
-                customWebpackConfigFile,
-                webpackResourceFiles,
-                entryPointFile,
-                mode.exportedName,
-                extraArgs,
-                webpackMode,
-                log
-              )
-              .file)
 
+          Webpack.bundleLibraries(
+            emitSourceMaps,
+            generatedWebpackConfigFile,
+            customWebpackConfigFile,
+            webpackResourceFiles,
+            entryPointFile,
+            mode.exportedName,
+            extraArgs,
+            webpackMode,
+            log
+          ).cached
         }
-      cachedActionFunction(monitoredFiles.to[Set])
-      generatedWebpackConfigFile.asLibrary(None)
+      val cached = cachedActionFunction(monitoredFiles.to[Set])
```

Comment:
Hmm, ok anyway `cachedActionFunction` can not return a `ListSet`… It can only return a `Set`.

Reply:
Right, we are limited by `FileFunction`'s API